test/libsolidity/syntaxTests/events/inheritance_adds_anonymous.sol

```diff
@@ -0,0 +1,8 @@
+contract A {
+    event X();
+}
+contract B is A {
+    event X() anonymous;
+}
+// ----
+// DeclarationError 5883: (52-72): Event with same name and arguments defined twice.
```

Comment:
Actually this would also clash with `event Y() anonymous`, but I think we can ignore that.

Reply:
Why would it clash with Y anonymous?